resteasy-rxjava2/pom.xml

```diff
@@ -63,6 +63,15 @@
         <groupId>org.jboss.logging</groupId>
         <artifactId>jboss-logging-processor</artifactId>
     </dependency>
+    <dependency>
+        <groupId>org.jboss.resteasy</groupId>
+        <artifactId>resteasy-reactive-context</artifactId>
```

Comment:
Just cosmetic comment, this should probably go up to other `org.jboss.resteasy` dependencies

deps should be ordered alphabetically by GA(V)